src/EnergyPlus/StandardRatings.cc

```diff
@@ -4508,7 +4508,7 @@ namespace StandardRatings {
                                 p_int, q_int, q_low, bl, n, Q_E_Int(spnum), q_full, P_E_Int(spnum), p_full, p_low);
                             speedsUsed.push_back(spnum);
                             goto SpeedLoop3_exit;
-                        } else if (bl < q_full) {
+                        } else { // bl < q_full
```

Comment:
[src/EnergyPlus/StandardRatings.cc:4511]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.